src/main/java/uk/gov/ons/census/casesvc/service/InvalidAddressService.java

```diff
@@ -21,18 +22,24 @@ public InvalidAddressService(CaseService caseService, EventLogger eventLogger) {
   }
 
   public void processMessage(ResponseManagementEvent invalidAddressEvent) {
-    // This check shouldn't be needed, but because several different shape messages are getting
-    // published to the same topic according to the event dictionary, let's keep this check in
-    // place so that we know we need to do something about it
-    if (invalidAddressEvent.getEvent().getType() != EventTypeDTO.ADDRESS_NOT_VALID) {
-      throw new RuntimeException(
-          String.format("Event Type '%s' is invalid!", invalidAddressEvent.getEvent().getType()));
+    EventTypeDTO eventType = invalidAddressEvent.getEvent().getType();
+
+    if (eventType.equals(EventTypeDTO.ADDRESS_MODIFIED)) {
+      eventLogger.logCaseEvent(
+          null,
+          invalidAddressEvent.getEvent().getDateTime(),
+          String.format("Consumed event type '%s'", eventType),
```

Comment:
I would say `"Address modified"` is the event description, even though we're not processing it properly

Reply:
done